src/lib/events.js

```diff
@@ -98,10 +120,24 @@ var Events = {
         /*
          * Call all the handlers except the last one.
          */
-        for(var i = 0; i < handlers.length; i++) {
+        for(i = 0; i < handlers.length; i++) {
             handlers[i](data);
         }
 
+        /* Do the same as for external-facing events, except trigger the same
+         * events on the internal handlers. This does *not* affect the return
+         * value. It simply mirrors triggers internally so that there's no
+         * conflict with external user-defined events when plotly manages
+         * events internally.
+         */
+        var internalHandlers = plotObj._internalEv._events[event];
```

Comment:
Perhaps not. I didn't fully understand the use-case of this. It appeared that it was for backward compatibility with earlier jquery-based events, but yeah, perhaps not.